### PR TITLE
Python laser geometry

### PR DESCRIPTION
**Pull request by efernandez**
Contributes a Python port for the simple laser projection, including a test

This PR can be also sync'd upstream with indigo-devel

### CODE REVIEW

**Comment by efernandez** [comment]
This addresses this ticket: https://code.ros.org/trac/ros-pkg/ticket/2867
(linked here: http://wiki.ros.org/laser_geometry)

File: src/laser_geometry/laser_geometry.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
+
```
Comment (vrabaud) [inline src/laser_geometry/laser_geometry.py:29]: please add a license

**Comment by vrabaud** [comment]
Hi, I was actually wondering what you'd think about this solution: wrap the C++ functionality through Boost Python like I did for cv_bridge:
https://github.com/ros-perception/vision_opencv/blob/indigo/cv_bridge/src/module.cpp
That would enable having only one code to maintain (C++). I could try to code that after ICRA.

**Comment by efernandez** [comment]
@vrabaud I've just added a license on top of the file.

Regarding Boost Python, if the solution is simple and clean, I like the fact that we won't have to maintain the C++ and Python code separately. On the other hand, I don't think the API and functionality will be extended so much, and IMHO the C++ could be simplify because it seems like there's a lot of redundancy in the code used for `PointCloud` and `PointCloud2`.

**Comment by vrabaud** [comment]
ok, I am merging this for now. I'll check if there is an easy way to use Boost Python here. I'll backport that to Indigo too. Thx for your good work !

**Pull request merged**